docs/gettingstarted/docker-compose-MacOS.md

```diff
@@ -1,72 +1,56 @@
 
 # Install Docker Desktop for MacOS
 
-Refer to [Installation](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+Refer to your Docker [documentation](https://docs.docker.com) to set up your Docker Desktop for MacOS. 
```

Comment:
can we preserve the link to docker-ce-desktop-mac?

Reply:
Generally we try to avoid any third party links, but when we do include them we link only to the general site, this is partly becuase we have no control over their docs, they might reorganize or change the links and there is no way we would know. We could say something like "Refer to the "MacOS" section of docker.com for more information." ?

Reply:
@jenworthington I like this proposition, thank you

Reply:
done:)